src/components/dashboard/RecoveryHeader/index.tsx

```diff
@@ -46,11 +48,33 @@ export function _RecoveryHeader({
   return null
 }
 
+export function _useIsProposalInProgress(): boolean {
+  const [isProposalSubmitting, setIsProposalSubmitting] = useState(false)
+  const queue = useRecoveryQueue()
+
+  useEffect(() => {
+    const unsubFns = Object.values(RecoveryEvent).map((event) =>
+      recoverySubscribe(event, (detail) => {
+        const isValidating = event === RecoveryEvent.EXECUTING || event === RecoveryEvent.PROCESSING
```

Comment:
Don't we have to ignore all events with `detail.eventType !== RecoveryEventType.PROPOSAL`?

Reply:
Adjusted in 03398c6976df9d148a727b5e15e8ba09feb7309f.